src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -442,6 +471,15 @@ public void partitionTest() throws Exception {
         }
     }
 
+    public com.google.cloud.bigquery.Dataset bigQueryDataset(Dataset dataset) {
+        return bigQueryPdao.bigQueryProjectForDataset(dataset)
+            .getBigQuery().getDataset(bigQueryPdao.prefixName(dataset.getName()));
+    }
+
+    public com.google.cloud.bigquery.Dataset bigQuerySnapshot(Dataset dataset, String datasetName) {
```

Comment:
I think changing renaming `datasetName` to something like `bigqueryDatasetName` would make this clearer since it's easy to confuse that with the data repo dataset name.